lib/components/src/blocks/ArgsTable/TabbedArgsTable.tsx

```diff
@@ -19,11 +20,9 @@ export const TabbedArgsTable: FC<TabbedArgsTableProps> = ({ tabs, ...props }) =>
       {entries.map((entry) => {
         const [label, table] = entry;
         const id = `prop_table_div_${label}`;
-        return (
+        return ({ active }: { active: boolean }) => (
           <div key={id} id={id} title={label}>
-            {({ active }: { active: boolean }) =>
-              active ? <ArgsTable key={`prop_table_${label}`} {...table} {...props} /> : null
-            }
+            {active ? <ArgsTable key={`prop_table_${label}`} {...table} {...props} /> : null}
           </div>
```

Comment:
not sure how to test this, but the types were wrong so I figured this is what the intent was.

Reply:
I think the types are incorrect here, chromatic picked up changes that don't look right.

Reply:
@lifeiscontent could you revert these runtime changes, and find another method for making the types fit? 👏

Reply:
@ndelangen do you know if the `div` element is always a `div`? or can it be any element? after that, I'll see if I can create a type that works for it, but I've never seen anything like that before as its telling a child element about its props which is pretty odd.

Reply:
I'm pretty sure the type of element `div` or otherwise doesn't matter, no.